### PR TITLE
Added EM Process Queue handler and Stage Transition 

### PR DESCRIPTION
**Pull request by bharathappali**
This PR is based on #280 can be reviewed once #280 is merged

Signed-off-by: bharathappali <abharath@redhat.com>

### CODE REVIEW

**Comment by bharathappali** [comment]
Description: 

As #280 adds the endpoints to receive the config we need a queue structure to push the transitions to be done so that the Stage Processor can read from the queue and perform the transition. This PR adds the Process Queues and Stage Transition classes which will be pushed into the queue for the Stage Processor to read

**Comment by bharathappali** [comment]
@dinogun This PR is ready to be reviewed as #280 has been merged

File: src/main/java/com/autotune/experimentManager/data/EMStageProcessQueue.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 Red Hat, IBM Corporation and others.
```
Comment (dinogun) [inline src/main/java/com/autotune/experimentManager/data/EMStageProcessQueue.java:2]: Should be 2022, 2022. Same for other files

File: src/main/java/com/autotune/experimentManager/data/EMStageProcessQueue.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+
+package com.autotune.experimentManager.data;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+/*
```
Comment (dinogun) [inline src/main/java/com/autotune/experimentManager/data/EMStageProcessQueue.java:22]: Comment should be in javadoc format. Same for other files too

**Review by dinogun — CHANGES_REQUESTED** [review]

File: src/main/java/com/autotune/experimentManager/data/EMStageProcessQueue.java (hunk 2 above)
Comment (bharathappali) [reply src/main/java/com/autotune/experimentManager/data/EMStageProcessQueue.java]: Updated comments with java doc format

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**